bin/slackin

```diff
@@ -6,7 +6,7 @@ var slackin = require('./../node').default;
 
 args
   .option(['p', 'port'], 'Port to listen on [$PORT or 3000]', require('hostenv').PORT || process.env.PORT || 3000)
-  .option(['h', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
+  .option(['H', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
```

Comment:
@alexpls One thing I could do on my side is add the ability to overwrite `-h` within the "args" package. What do you think about that? I feel like it's a good thing on one hand, but on the other, it confuses people because most CLIs are using `-h` for the usage information.